src/core/sys/posix/aio.d

```diff
@@ -0,0 +1,65 @@
+/**
+ * D header file to interface with the Linux aio API (http://man7.org/linux/man-pages/man7/aio.7.html).
```

Comment:
Any link to the posix standard that is not a Linux manpage?

Reply:
Will this suffice?
https://www.freebsd.org/cgi/man.cgi?query=aio&sektion=4&apropos=0&manpath=FreeBSD+11.1-RELEASE+and+Ports

Reply:
Also http://www2.fisica.unlp.edu.ar/electro/temas/p7/POSIX4.pdf , see page 277

Reply:
This one seems best: http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/aio.h.html

Reply:
I made the change myself. All, feel free to operate noncontroversial changes (spacing, better links) right on the spot - it takes only a bit longer than making the request and accelerates the cycle greatly.